.github/workflows/release.yml

```
@@ -5,11 +5,14 @@ on:
 jobs:
   ci:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16, 18]
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v2
         with:
-          node-version: "16"
+          node-version: ${{ matrix.node-version }}
```

Isn't one node version enough for release?
I would suggest using the one from `.node-version`

```suggestion
          node-version-file: ".node-version"
```

And removing line 8-10.

---

@clintonmedbery suggested the matrix for testing. Clint do you have any thoughts on the benefits of the matrix vs a single version?

---

This is the `release` workflow, not the testing workflow :)

---

Yeah I like the idea of testing on multiple versions (if that is what we want to do) but I think the release only needs one version.